tests/unit/test_registries.py

```diff
@@ -0,0 +1,81 @@
+#pylint: skip-file
+
+import unittest
+from Atomic.util import is_backend_available, load_registries_from_yaml, get_registries
+import json
+import subprocess
+
+no_mock = True
+try:
+    from unittest.mock import MagicMock, patch
+    no_mock = False
+except ImportError:
+    try:
+        from mock import MagicMock, patch
+        no_mock = False
+    except ImportError:
+        # Mock is already set to False
+        pass
+
+
+@unittest.skipIf(no_mock, "Mock not found")
+class TestRegistriesFromYAML(unittest.TestCase):
+
+    def compare_list_of_dicts(self, results, answer):
+        if not isinstance(results, list) or not isinstance(answer, list):
+            raise AssertionError("Results must always be of type list")
+        if len(results) != len(answer):
+            raise AssertionError("Length of the lists differ.")
+        for registry in results:
+            self.assertDictEqual(registry, next(item for item in answer if item["hostname"] == registry['hostname']))
+
+    def test_no_registries(self):
+        with patch('Atomic.util.is_backend_available') as dockerd:
+            dockerd.return_value = False
+            with patch('Atomic.util.registries_tool_path') as reg_path:
+                reg_path.return_value = "/usr/libexec/registries"
```

Comment:
It will be a subpackage of atomic last we spoke about it.  I had planned to exercise it via an integration test but not until it is in the spec.  These unit tests just test the path through get_registries to make sure the outputs don't regress.